src/app/components/room/room.service.js

```diff
@@ -144,7 +147,14 @@
             ActionService.enterRoom(msg.id, username, connection);
             // Step 3. Establish WebRTC connection with the Janus server
             // Step 4a (parallel with 4b). Publish our feed on server
+
             connection.publish({
+              success: function() {
```

Comment:
It would be nice to explain the user what is happening. Showing a notification "you start muted because bla bla" would be a legit usage for the `success` callback (not saying it's the correct place to put it, but at least it would not be abusing it).